src/Framework/FieldsAPI/Concerns/TapNode.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Give\Framework\FieldsAPI\Concerns;
+
+/**
+ * @unreleased
+ */
+trait TapNode
```

Comment:
@kjohnson Why not just add this method to the `Node` class and let inheritance do its thing? Why apply the trait to each child class separately?

Reply:
`Node` is an interface. `Element`, `Field`, and `Group` are the base concrete implementations.

```php
interface Node extends JsonSerializable {
     ...
}
```

Reply:
Oh. That would explain that. 😁

Reply:
Yeah, caught me off guard. "What do I do now???"